### ISSUE TITLE
Add 'Single Source of Truth'

### ISSUE THREAD

**Issue by dwmkerr** [dwmkerr/hacker-laws#223]
Relates to DRY, see also:

#207 

https://en.wikipedia.org/wiki/Single_source_of_truth

**Comment by guettli** [comment]
Yes, this single-source-of-truth is important. For me it is especially important for documentation.

Every term should have a canonical URL.

Related: https://github.com/guettli/programming-guidelines/blob/master/README.md#canonical-docs
